### ISSUE TITLE
futurerestore does not build or install 

### ISSUE THREAD

**Issue by maenpaa24** [stek29/homebrew-idevice#23]
Futurerestore does not install. The log throws the following:

```
configure: error: Package requirements (libirecovery >= 0.2.0) were not met:
No package 'libirecovery' found
configure: error: ./configure failed for external/idevicerestore
```

libirecovery is actually installed (stable 1.0.0 (bottled), HEAD) and homebrew is able to install from source the idevicerestore formula from this repo.

I think it is related to how dependencies are managed by homebrew for the submodules in external, that is, external/idevicerestore and external/tsschecker. I mean, the superenv in which homebrew builds stuff does not see libirecovery when building externals, maybe because their dependencies are not specified in the ruby formula file. I do not master homebrew, so it is just my guess.

**Comment by stek29** [comment]
I give up until futurerestore/futurerestore#75 is fixed

**Comment by stek29** [comment]
whole state of futurerestore is a mess – tihmstar's original repos haven't been updated and depend on old versions of his own libraries, there are tons of different forks and all of them just don't seem to care about proper versioning and making their projects friendly for package management

**Comment by stek29** [comment]
fixed in #45

**Comment by stek29** [comment]
(I haven't actually tested the futurerestore that was build though)
